### PR TITLE
Site improvements

### PR DESCRIPTION
**Pull request by mg8973**
I added a responsive design element to some of the pages that include large images so they adjust to every screen size.

### CODE REVIEW

File: _includes/items.html [hunk 1]
```diff
@@ -3,13 +3,6 @@
 {% for post in posts reversed %}
   {% unless post.tags contains 'switch' %}
     {% unless post.tags contains 'stick_to_windows' %}
-      <a href="{{ post.url }}">
```
Comment (lmanul) [inline _includes/items.html]: Why this change? This removes most of what is on the home page. Is this intentional?

Comment (lmanul) [reply _includes/items.html]: Okay I think I made a change to resolve the problem you saw. Could you update your pull request to revert this file but leave the other changes? Thanks!

**Review by lmanul — CHANGES_REQUESTED** [review]

File: _includes/items.html (hunk 1 above)
Comment (mg8973) [reply _includes/items.html]: Sorry, I’m new to GitHub, how I do I revert this file on desktop?

Comment (mg8973) [reply _includes/items.html]: Never mind, I figured it out, I think the pull request is good now.

**Review by lmanul — APPROVED** [review]
Thank you for fixing this!

**Comment by mg8973** [comment]
I may have to modify this, because it also changes the size of the flag icons, which now renders the website unusable. ☹

**Comment by lmanul** [comment]
No worries, should be all good now :-)
